.env.local.example

```diff
@@ -1,3 +1,2 @@
 # create local copy of this file and rename to .env.local
 # make sure to add .env.local to .gitignore!!
-MONGO_URI={mongo-uri-here}
```

Comment:
One more small thing, could you keep the `MONGO_URI` placeholder as is so we can use it as a reference in the future? Thanks!